src/content/core/namespaces.mdx

```diff
@@ -93,7 +93,6 @@ These recommendations might include increasing or decreasing CPU and memory requ
     alt="share a namespace"
     width="850"
   />
--->
```

Comment:
@rlamana @pchico83 Do you think these screenshots work ok here? Should we also add screenshots for: 1) what it looks like when RM is not installed, 2) what it looks like when RM is installed, but not enabled, 3) what it looks like when RM is installed and has recommendations?

Reply:
As long as we explain those states (not installed, installed and automatic, installed and manual), I don't think it is necessary to show more screenshot for each state. They look very similar, and the "not installed" one is self-explanatory once the user is in there.